.github/workflows/build-plugin/action.yml

```diff
@@ -22,7 +22,13 @@ runs:
                 echo "PACKAGE_VERSION is empty string or undefined"
                 exit 1
             fi
-            bash ${{ inputs.UPDATE_VERSIONS_SCRIPT_PATH }}
+
+            if [ "${{ inputs.UPDATE_VERSIONS_SCRIPT_PATH }}" != "" ];
```

Comment:
Add a "TODO" comment to remove this when we stop using the cloud publish

Reply:
Done